src/shared/Microsoft.Git.CredentialManager/Trace.cs

```diff
@@ -337,4 +338,28 @@ private static string FormatText(string message, string filePath, int lineNumber
             return text;
         }
     }
+
+    public static class TraceExtensions
+    {
+        public static void WriteProxyInfo(this ITrace trace, IWebProxy proxy)
+        {
+            var uri = proxy.GetProxy(new Uri("https://example.com"));
```

Comment:
Can you explain what this function is doing?  Sample output in comments would be helpful.